examples/DynamicCantileverCase/dynamic_cantilever_helper.py

```diff
@@ -0,0 +1,96 @@
+import numpy as np
+
+
+class DynamicCantileverVibration:
+    def __init__(
+        self,
+        base_length,
+        base_area,
+        moment_of_inertia,
+        youngs_modulus,
+        density,
+        mode,
+        end_velocity=0.0,
+    ):
+        self.base_length = base_length
+        self.end_velocity = end_velocity
+
+        assert isinstance(mode, int)
+        assert mode >= 0
+        assert mode < 5
+        self.mode = mode
+
+        # First four roots to
+        # cosh(beta * base_length) * cos(beta * base_length) + 1 = 0
+        #
+        # Beta values are used to determine natural frequencies of cantilever beams
+        # based on dynamic beam equations
+        # omega = beta^2 * sqrt(E * I / mu)
+        #
+        # For details, refer to
+        # https://en.wikipedia.org/wiki/Euler%E2%80%93Bernoulli_beam_theory#Dynamic_beam_equation
+        # or
+        # Han et al (1998), Dynamics of Transversely Vibrating Beams
+        # Using Four Engineering Theories
```

Comment:
This explanation is all good, but lets do this in proper class docstring. You can see this example https://github.com/GazzolaLab/PyElastica/blob/master/elastica/external_forces.py#L273-L295

Reply:
[144e2af](https://github.com/GazzolaLab/PyElastica/pull/173/commits/144e2afdb705bd31365ba618f447211d23619aed)